diff --git a/src/libkirigami/units.cpp b/src/libkirigami/units.cpp
index e7c0a31..6b78788 100644
--- a/src/libkirigami/units.cpp
+++ b/src/libkirigami/units.cpp
@@ -68,7 +68,7 @@ public:
         // Cache font so we don't have to go through QVariant and property every time
         , fontMetrics(QFontMetricsF(QGuiApplication::font()))
         , gridUnit(fontMetrics.height())
-        , smallSpacing(std::floor(gridUnit / 4))
+        , smallSpacing(std::floor(gridUnit / 2))
         , mediumSpacing(std::round(smallSpacing * 1.5))
         , largeSpacing(smallSpacing * 2)
         , veryLongDuration(400)
